examples/bridge/src/application/mod.rs

```diff
@@ -23,14 +22,9 @@ pub struct Config<H: Hasher, Si: Sink, St: Stream, P: Array> {
     /// Hashing scheme to use.
     pub hasher: H,
 
-    /// Prover used to decode opaque proofs from consensus.
-    pub prover: Prover<H::Digest>,
-
-    pub other_prover: Prover<H::Digest>,
-
+    pub namespace: Vec<u8>,
     pub identity: poly::Public,
-
-    pub other_network: group::Public,
+    pub other_identity: group::Public,
```

Comment:
seems weird that these aren't the same type. Is this intended?

Reply:
Yeah, that was intended but can understand the confusion.

Reply:
I'll change to `other_public` (including the CLI args)